.github/workflows/pr-test.yml

```diff
@@ -208,6 +208,15 @@ jobs:
     env:
       RUNNER_LABELS: 1-gpu-runner
     steps:
+      - name: Kill lingering processes and cleanup workspace
+        run: |
+          pkill -f "python.*sglang" || true
```

Comment:
Can we move this command to a new stage after Run Test so that this script can stored in a file in the repo instead?

Also do we need to run rm -rf "$GITHUB_WORKSPACE"/* || true? IIUC this will remove the code repo as well might slow down the code checkout time, but I'm not sure about the performance implication